app/src/main/AndroidManifest.xml

```diff
@@ -1,177 +1,187 @@
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
-    <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES"/>
-    <uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES"/>
-    <!-- For GcmTaskService -->
-    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+    <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES" />
+    <uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES" /> <!-- For GcmTaskService -->
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
 
-    <uses-feature android:name="android.hardware.camera" android:required="false"/>
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
 
     <application
-            android:allowBackup="false"
-            android:name=".app.FlowApp"
-            android:hasCode="true"
-            android:icon="@mipmap/ic_launcher"
-            android:label="@string/app_name"
-            android:theme="@style/FlowAppTheme"
-            android:usesCleartextTraffic="true"
-            tools:ignore="GoogleAppIndexingWarning">
+        android:name=".app.FlowApp"
+        android:allowBackup="false"
+        android:hasCode="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:theme="@style/FlowAppTheme"
+        android:usesCleartextTraffic="true"
+        tools:ignore="GoogleAppIndexingWarning">
+        <activity
+            android:name=".presentation.datapoints.map.offline.list.OfflineAreasListActivity"
+            android:label="@string/offline_maps_list_title"
+            android:parentActivityName=".activity.SurveyActivity">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value="org.akvo.flow.activity.SurveyActivity" />
+        </activity>
+        <activity
+            android:name=".presentation.datapoints.map.offline.selection.OfflineMapDownloadActivity"
+            android:label="@string/offline_map_activity_title" />
+
         <meta-data
-                android:name="io.fabric.ApiKey"
-                android:value="89dc21727e76a10511ecda549cab1b2d3aff2ec7"/>
+            android:name="io.fabric.ApiKey"
+            android:value="89dc21727e76a10511ecda549cab1b2d3aff2ec7" />
```

Comment:
its not problematic to have this key in here I guess no? i.e. security issue

Reply:
That's how they tell you to do it in the documentation, we could extract it to string resources but in the end it does not change much. This key is used to send betas to fabric, not sure what else anyone could do with it.